cmd/jira-lifecycle-plugin/server.go

```diff
@@ -2311,6 +2386,103 @@ func handleClose(e event, gc githubClient, jc jiraclient.Client, options JiraBra
 	return nil
 }
 
+func handleVerification(e event, ghc githubClient, inserter BigQueryInserter, log *logrus.Entry) error {
+	comment := e.comment(ghc)
+	if len(e.verifyLater) > 0 && len(e.verify) > 0 {
+		return comment("The `/verified` and `/verified later` commands cannot be used in the same comment.")
+	}
+	msg := ""
+	if len(e.verifyLater) > 0 {
+		// make sure the PR is not already marked as verified
+		var existingLabel bool
+		if prLabels, err := ghc.GetIssueLabels(e.org, e.repo, e.number); err != nil {
+			log.WithError(err).Warn("Could not list labels on PR")
+			return comment("Failed to check labels for this PR. Please try again.")
+		} else {
+			for _, label := range prLabels {
+				if label.Name == labels.Verified {
+					return comment("PR is already has `verified` label. Cannot change PR to `verify-later`")
+				}
+				if label.Name == labels.VerifiedLater {
+					existingLabel = true
+				}
+			}
+		}
+		// only usernames can be verified-later
+		for _, reason := range e.verifyLater {
+			if !strings.HasPrefix(reason, "@") {
+				return comment("Only users can be targets for the `verified later` command.")
```

Comment:
Probably should have the forward slash since this is a reference to the command.